### PR TITLE
feat: Add more ACs for 0073-LIMN spec.

### PR DESCRIPTION
**Pull request by jgsbennett**
Some more ACs for additional scenarios covered by https://github.com/vegaprotocol/system-tests/pull/1013

### CODE REVIEW

File: protocol/0073-LIMN-limited_network_life.md [hunk 1]
```diff
@@ -299,3 +299,21 @@ If for `network.checkpoint.timeElapsedBetweenCheckpoints` the value is set to `0
...
 1. Load the checkpoint into a new network
 1. Assert that at every epoch, the recurring transfers to the reward pool continues to happen, and that the funds are properly being distributed to the delegator
+
+## Test case 12:
+Enacted, listed ERC20 asset is remembered in checkpoint (<a name="0073-LIMN-023" href="#0073-LIMN-023">0073-LIMN-023</a>)
+An ERC20 asset loaded from checkpoint can be used in a market loaded from a checkpoint (<a name="0073-LIMN-024" href="#0073-LIMN-024">0073-LIMN-024</a>)
+An ERC20 asset loaded from checkpoint can be updated (<a name="0073-LIMN-025" href="#0073-LIMN-025">0073-LIMN-025</a>)
+An ERC20 asset loaded from checkpoint can be used in newly proposed markets (<a name="0073-LIMN-026" href="#0073-LIMN-026">0073-LIMN-026</a>)
+Can deposit and withdraw funds to/from ERC20 asset loaded from checkpoint (<a name="0073-LIMN-027" href="#0073-LIMN-027">0073-LIMN-027</a>)
+Can update an ERC20 asset loaded from checkpoint (<a name="0073-LIMN-028" href="#0073-LIMN-028">0073-LIMN-028</a>)
```
Comment (davidsiska-vega) [inline protocol/0073-LIMN-limited_network_life.md]: What's the difference between 028 and 025?

File: protocol/0073-LIMN-limited_network_life.md [hunk 2]
```diff
@@ -299,3 +299,21 @@ If for `network.checkpoint.timeElapsedBetweenCheckpoints` the value is set to `0
...
+
+1. Propose a valid ERC20 asset.
```
Comment (davidsiska-vega) [inline protocol/0073-LIMN-limited_network_life.md:310]: What AC code does this playbook have?

Comment (jgsbennett) [reply protocol/0073-LIMN-limited_network_life.md:310]: I was tying one case to have several AC codes (i.e: All of the above) as something of a trade off, the cases can be quite time consuming to execute, so combining saved a decent chunk of time.

File: protocol/0073-LIMN-limited_network_life.md (hunk 1 above)
Comment (jgsbennett) [reply protocol/0073-LIMN-limited_network_life.md]: There is none, currently. I either forgot I had it, or intended it to be something else (I plan to add more cases). I'll tidy it up.

**Review by davidsiska-vega — APPROVED** [review]

**Pull request merged**